init.sh

```diff
@@ -88,6 +88,7 @@ cd "$DIR_LIB_GEN" && [ ! -f "$REPLACECALL_JAR" ] \
 echo
 echo "Setting up Gradle dependencies ... "
 GRADLE_ZIP=defects4j-gradle.zip
+GRADLE_DEPS_ZIP=gradle_deps.zip
```

Comment:
Should we merge the gradle distributions and all gradle dependencies -- what are the pros and cons?
If we keep them separated, I suggest the following names:
* `defects4j-gradle-dists.zip`
* `defects4j-gradle-deps.zip`

Reply:
I suspect the number of Gradle version supported by D4J (i.e., `defects4j-gradle-dists.zip`) won't change very often. So, yes, lets keem them in separated. You can find the `defects4j-gradle-deps.zip` in [here](https://drive.google.com/open?id=1bYz4wNF3jEhxpY0Hnu-8aSzrtIOWm4qL).